openedx/core/djangoapps/theming/test_util.py

```diff
@@ -64,7 +53,7 @@ def with_is_edx_domain(is_edx_domain):
     def _decorator(func):                       # pylint: disable=missing-docstring
         if is_edx_domain:
             # This applies @with_comprehensive_theme to the func.
-            func = with_comprehensive_theme(EDX_THEME_DIR)(func)
+            func = with_comprehensive_theme('edx.org')(func)
```

Comment:
@andy-armstrong above tests were added for the verification of the removal of `IS_EDX_DOMAIN` flag. That is why 'edx.org' is used inside tests. 
I have also updated these decorators to remove hardcoded edx.org from these decorators and moved them into their corresponding tests, kindly take a look at the updated code.